content/en/docs/tasks/debug-application-cluster/window.md

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
Super nit but should the file name be windows.md instead of window.md?

Reply:
updated!